.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,30 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
```

Comment:
Is this empty line here on purpose?
(Same question for the other issue template)

Reply:
Yes, they are. I don't know the exact reason behind it but it's how GitHub generates them.

Reply:
Got it.